endpoint/models/endpoint_mixin.py

```diff
@@ -184,14 +187,18 @@ def _get_handler(self):
                 _("Missing handler for exec mode %s") % self.exec_mode
             )
 
-    def _handle_request(self, request):
+    def _handle_request(self, request, params=None):
         # Switch user for the whole process
         self_with_user = self
         if self.exec_as_user_id:
             self_with_user = self.with_user(user=self.exec_as_user_id)
         handler = self_with_user._get_handler()
         try:
-            res = handler(request)
+            # In case the handler does not support params
```

Comment:
I doubt we have any other handler ATM but just in case... ok, let's keep this but I would add a separated commit for backward compat that we can easily drop on migration. 
Something like:

```
[imp] endpoint: support querystring params
[imp] endpoint: backward compat for querystring params

Old handlers might not accept this new kw args
Trash this commit in future versions. Eg: v18.
```